src/tenants/planet/LeaderBoard/components/QuarterlyReports.tsx

```diff
@@ -0,0 +1,100 @@
+  const [baseValue, setBaseValue] = React.useState(0);
+  React.useEffect(() => {
+    async function fetchReports() {
+      const newreports = await getRequest('/app/reports/donationsQuarterly');
+      const maxValue = newreports.reduce((a, b) => a.treesTotal > b.treesTotal ? a : b).treesTotal;
+      setBaseValue(maxValue);
+      setreports(newreports);
+    }
+    fetchReports();
+  }, []);
+
+  return ready ? (
+    <div className={styles.reportsSection}>
+      <div className={styles.reportsTitle}>
+        <h2>{t('leaderboard:quarterlyReportsTitle')}</h2>
+      </div>
+
+      <div className={styles.allReports}>
+        {reports && reports.length > 0 ? (
+          reports.map((report, index) => {
+            return (
+              <div key={index} className={styles.singleQuarter}>
+                <p className={styles.quarterTitle}>
+                  {index === 0 ? (
+                    t('leaderboard:current')
+                  ) : (
+                      `Q${report.quarter} ${report.year}`
+                    )}
+
+                </p>
+                {report.treesDonated && report.treesDonated.length > 0 ? (
+                  <div className={styles.reportContainer}>
+                    {report.treesDonated.map((project, index) => {
+                      const opacity = (100 - (index * 14)) / 100;
+                      return (
+                        <div 
+                          onClick={() => {
+                            project.id !== 0 ?
+                            router.push('/[p]', `/${project.id}`, {
+                              shallow: true,
+                            }) : {}
+                          }} 
+                          key={index} 
+                          className={styles.projectSegmentContainer} 
+                          style={{ width: `${(project.trees / baseValue) * 100}%` }}>
+                          
+                          <div className={styles.projectSegment} style={{ opacity: opacity }}/>
+                          <div className={styles.projectName}>{project.name} {" "}
+                            <span>{t('leaderboard:treesTotal', {
+                              count: getFormattedNumber(i18n.language, project.trees),
+                            })}</span>
+                          </div>
+                        </div>
+                      )
+                    })}
+                  </div>
+                ) : null}
+                <p className={styles.totalTrees}>
+                  {t('leaderboard:treesTotal', {
+                    count: getFormattedNumber(i18n.language, report.treesTotal),
+                  })}
+                </p>
+              </div>
+            )
+          })
+        ) : null}
+
+      </div>
+
+
+      <p className={styles.reportDescription}>{t('leaderboard:reportDescription')}</p>
+
+      <a rel="noreferrer" href="https://www.plant-for-the-planet.org/en/donation" target="_blank">
```

Comment:
Can we localize this? For German this link is https://www.plant-for-the-planet.org/de/spenden and I do not know about other supported languages :-( So just knowing the right locale won't work. We would have to provide an associate array with localized links for each of our supported locales.

Reply:
In our footer at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/c815dd9de987c26279917ea71a6d1dc9c604da9d/src/features/common/Layout/Footer/index.tsx#L77 we use https://a.plant-for-the-planet.org/${locale}/donation - can't we use that link?

Reply:
Yes we can, that should redirect you to the correct donation page.